build.gradle

```diff
@@ -32,7 +31,6 @@ allprojects {
     apply plugin: 'java'
     apply plugin: 'jacoco'
     apply plugin: 'eclipse'
-    apply plugin: 'findbugs'
```

Comment:
findbugs plugin was removed in Gradle 6
https://docs.gradle.org/6.0-rc-3/userguide/upgrading_version_5.html#the_findbugs_plugin_has_been_removed